client/src/components/navigation/Navbar.js

```diff
@@ -1,30 +1,47 @@
-import React from 'react';
+import React from "react";
 
-import makeStyles from '@material-ui/core/styles/makeStyles';
-import Box from '@material-ui/core/Box';
-import Typography from '@material-ui/core/Typography';
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import Box from "@material-ui/core/Box";
+import Tab from "@material-ui/core/Tab";
+import Tabs from "@material-ui/core/Tabs";
 
 const useStyles = makeStyles({
   root: {
-    backgroundColor: 'white',
-    padding: '2rem',
-    marginBottom: '2rem',
+    backgroundColor: "white",
+    padding: "1rem ",
+    marginBottom: "2rem",
   },
   text: {
-    textAlign: 'center',
+    textAlign: "center",
   },
 });
 
+// TODO: Add functionality for page changes when react-router is implemented
 const Navbar = () => {
   const classes = useStyles();
+  const [value, setValue] = React.useState("Home");
```

Comment:
Since this is just a placeholder, this is ok for now. 

When we have React Router set up, we should make a constants file to store all the possible page names :)